java/S3ParquetSink/README.md

```diff
@@ -60,27 +99,102 @@ Flink will [natively and efficiently serialize and deserialize](https://nightlie
 
-1. `org.apache.flink:flink-connector-files:1.15.2` - File Sink Connector
-2. `org.apache.flink:flink-s3-fs-hadoop:1.15.2` - Support for writing to Amazon S3.
-3. `org.apache.flink:flink-avro:1.15.2` - Support for using Avro with Flink .
+1. `org.apache.flink:flink-connector-files:1.15.4` - File Sink Connector
+2. `org.apache.flink:flink-s3-fs-hadoop:1.15.4` - Support for writing to Amazon S3.
+3. `org.apache.flink:flink-avro:1.15.4` - Support for using Avro with Flink .
 
 
 ### Running in Intellij
 
 For running this example locally, you will need to make additional changes in the configuration. 
-The Hadoop Dependencies are not capable of capturing the AWS Credentials locally in Intellij, for which you will need to provide your Access Key and Secret Access Key in a core-site.xml file.
 
 Steps to follow:
-* Update core-site.xml file with AWS Credentials that have permissions to consume data from a Kinesis Data Stream and write data into Amazon S3
-* Update the flink-conf.yaml and add the path to the core-site.xml file in `fs.hdfs.hadoopconf`
-* You will need to add the following environmental variable before starting the Flink job, please update with path to your flink-conf.file
-  * FLINK_CONF_DIR= `Path to flink-conf.file`
-
-To start the Flink job in IntelliJ edit the Run/Debug configuration enabling 'Add dependencies with "provided" scope to
-the classpath'.
-
-
+* Update `resources/flink-application-properties-dev.json`. Add Kinesis Data Stream Name , Stream Region and S3 path to write the data to.
+```shell
+## Example
+[
+  {
+    "PropertyGroupId": "FlinkApplicationProperties",
+    "PropertyMap": {
+      "input.stream": "stream-input",
+      "stream.region": "ap-south-1",
+      "s3.path": "s3://aksh-flink-sink/flink/msf/"
+    }
+  }
+]
+```
+* Execute with AWS user that has permissions to consume data from a Kinesis Data Stream and write data into Amazon S3.
+* To start the Flink job in IntelliJ edit the Run/Debug configuration enabling 'Add dependencies with "provided" scope to
+  the classpath'.
+* Following is the screenshot of run configuration
+  ![Run Configuration](images/runConfiguration.png)
+
+
+
+## Running locally through Maven command line
+* Update `resources/flink-application-properties-dev.json`. Add Kinesis Data Stream Name , Stream Region and S3 path to write the data to.
+* Execute with AWS user that has permissions to consume data from a Kinesis Data Stream and write data into Amazon S3.
+* Execute following command from the project home directory -
+
+```
+ mvn clean compile exec:java  -Dexec.classpathScope="compile" \
+ -Dexec.mainClass="com.amazonaws.services.msf.StreamingJob" 
+
+```
+
+## Deploying using CloudFormation to Amazon Managed Service for Apache Flink
+### Pre-requisite
+1. Kinesis stream and S3 bucket. 
+2. Create subnets and security groups for the Flink application. If you are using private subnets , ensure that VPC endpoint for Kinesis and S3 is created.
```

Comment:
Same as per #29, VPC networking is not required

Reply:
Done. Made CloudFormation changes as well and tested.